### PR TITLE
Russian l10n

### PR DESCRIPTION
**Pull request by aviriel**

### CODE REVIEW

**Comment by Heziode** [comment]
Hi, 

I let your PR in pending the time I finish some additions that involve changes in the structure of the language files.

I tell you when it is done for so you can update your translation. ;)

**Comment by Heziode** [comment]
It is done, you can rebase and update your translation

**Comment by aviriel** [comment]
Thanks, will do it tomorrow

**Comment by Heziode** [comment]
Hi @aviriel,

I wait your changes before releasing the new version.

**Comment by aviriel** [comment]
Ok, it took more time than I expected. Give me few hours to review the translation and to push changes

**Comment by Heziode** [comment]
Ok no problem.

**Comment by aviriel** [comment]
Looks like that's all. You can merge if it's ok.

File: i18n/ru.json [hunk 1]
```diff
@@ -0,0 +1,298 @@
...
+                  "P3({{investment}})",
+                  "P4({{investment}})",
+                  "P5({{investment}})"
+               ]
+            }
+         },
+         "secure_position": {
+            "block_place": {
+               "title": "Обеспечение гарантии",
+               "description": {
```
Comment (Heziode) [inline i18n/ru.json]: From 99 to 102, this block has been removed. The description used instead is at line [290](https://github.com/FOE-Tools/FOE-Tools.github.io/pull/14/files#diff-8387edab8da18c7d7e2b04fb4e9c68aeR290).

**Comment by Heziode** [comment]
Thank you for this PR.

**Pull request merged**